### ISSUE TITLE
Not able to pass resources block to chart

### ISSUE THREAD

**Issue by kingnarmer** [hpe-storage/truenas-csp#47]
chart install fails when supplied resources block on values input. 

Error: 
``` Error: values don't meet the specifications of the schema(s) in the following chart(s):
│ truenas-csp:
│ - resources: Additional property requests is not allowed```


I think the vales.schema.json need to be updated for "resources" section. 

https://github.com/hpe-storage/truenas-csp/blob/1657b8f76aeed99a29e364f8027dccc4734b7a07/helm/charts/truenas-csp/values.schema.json#L344

**Comment by datamattsson** [comment]
Thanks for reporting this. I'll take a look when I'm back from vacation
next week.

**Comment by ajchiarello** [comment]
Is there any movement on this?

**Comment by datamattsson** [comment]
This has been fixed in  [v2.4.2](https://github.com/hpe-storage/truenas-csp/releases/tag/v2.4.2). Defaults are in [values.yaml](https://raw.githubusercontent.com/hpe-storage/truenas-csp/master/helm/charts/truenas-csp/values.yaml).
